modular_skyrat/modules/company_imports/code/objects/jarnsmiour/special_metals.dm

```diff
@@ -0,0 +1,117 @@
+// Completely made up materials to be sold in bar form by jarnsmiour in cargo, *should* be unobtainable otherwise
```

Comment:
what's the point of these, exactly?

Reply:
Likely for forging to make cool-looking items. It's seemingly a forging-related company anyways.

Reply:
stalkeros got it right on, they solely exist for making cool forging weapons out of and nothing else